server/routes/charts.js

```diff
@@ -264,7 +274,9 @@ router
         .collection(collections.charts)
         .insertOne({
           ...sourceChart,
+          title: `${sourceChart.title} (copy)`,
```

Comment:
sorry can you explain that a bit more? I'm not understanding what you mean.

Reply:
Yes sorry,  I think that if we make the title into a variable it would look cleaner and if there's feedback about changing the title it's simpler to edit and find.

`const title = `${sourceChart.title} (copy)` 
`
and then use that variable 
`.insertOne({
          ...sourceChart,
          title
})
what are your thoughts?

Reply:
I think since the variable wouldn't be used anywhere else this is fine for now. I like the explicitness where you can see exactly how the title is changed for duplicate charts instead of chasing down the variable.